cdap-hbase-compat-base/src/main/java/co/cask/cdap/data2/util/hbase/HBaseTableUtil.java

```diff
@@ -498,6 +498,12 @@ public void truncateTable(HBaseDDLExecutor ddlExecutor, TableId tableId) throws
     ddlExecutor.truncateTable(tableName.getNamespaceAsString(), tableName.getQualifierAsString());
   }
 
+  public void grantPrivileges(HBaseDDLExecutor ddlExecutor, TableId tableId,
+                              Map<String, String> privileges) throws IOException {
```

Comment:
we don't need to have this method. Caller can simply call `ddlExecutor.grantPersmissions`